cpp/ql/src/semmle/code/cpp/NameQualifiers.qll

```diff
@@ -99,8 +121,19 @@ class NameQualifiableElement extends Element, @namequalifiableelement {
 }
 
 /**
- * A C++ element that can qualify a name. For example, `N` in `N::f()`.  A
- * name-qualifying element is either a namespace or a user-defined type.
+ * A C++ element that can qualify a name. For example, the namespaces `A` and
+ * `A::B` and the class `A::C` in the following code:
+ * ```
+ * namespace A {
+ *   namespace B {
+ *     ...
+ *   }
+ * 
+ *   class C {
+ *     ...
+ *   };
+ * }
+ * ```
  */
```

Comment:
For completeness, I would also add a scoped enum:
``enum class E { e1, e2...};``

Reply:
Otherwise LGTM.

Reply:
Seems reasonable.  Done.